gulpfile.js

```diff
@@ -13,177 +13,178 @@ USWDS SASS GULPFILE
 
 const autoprefixer = require("autoprefixer");
 const csso = require("postcss-csso");
-const gulp = require("gulp");
-const pkg = require("./node_modules/uswds/package.json");
+const { src, dest, series, parallel, watch } = require("gulp");
+const pkg = require("../uswds/package.json").version;
 const postcss = require("gulp-postcss");
 const replace = require("gulp-replace");
 const sass = require("gulp-sass")(require("sass"));
 const sourcemaps = require("gulp-sourcemaps");
-const uswds = require("./node_modules/uswds-gulp/config/uswds");
 const del = require("del");
 const svgSprite = require("gulp-svg-sprite");
 const rename = require("gulp-rename");
+const log = console.log;
+const colorBlue = "\x1b[34m%s\x1b[0m";
+
 
 /*
 ----------------------------------------
 PATHS
 ----------------------------------------
-- All paths are relative to the
-  project root
-- Don't use a trailing `/` for path
-  names
-----------------------------------------
 */
+let paths = {
+  src: {
+    uswds: `node_modules/uswds/dist`,
+    sass: `node_modules/uswds/dist/scss/theme/**/**`,
+    fonts: `node_modules/uswds/dist/fonts/**/**`,
+    img: `node_modules/uswds/dist/img/**/**`,
+    js: `node_modules/uswds/dist/js/**/**`,
+  },
+  /**
+   * ? project paths
+   * - all paths are relative to the project root
+   * - don't use a trailing `/` for path
+   */
+  dist: {
+    sass: "./sass",
+    img: "./images",
+    fonts: "./fonts",
+    js: "./js",
+    css: "./css",
+  },
+}
 
-// Project Sass source directory
-const PROJECT_SASS_SRC = "./path/to/project/sass";
-
-// Images destination
-const IMG_DEST = "./path/to/images/destination";
-
-// Fonts destination
-const FONTS_DEST = "./path/to/fonts/destination";
-
-// Javascript destination
-const JS_DEST = "./path/to/js/destination";
 
-// Compiled CSS destination
-const CSS_DEST = "./path/to/css/destination";
+/*
+----------------------------------------
+TASKS
+----------------------------------------
+*/
 
-// Site CSS destination
-// Like the _site/assets/css directory in Jekyll, if necessary.
-// If using, uncomment line 106
-const SITE_CSS_DEST = "./path/to/site/css/destination";
+/*
+----------------------------------------
+USWDS specific tasks
+----------------------------------------
+*/
+const usaTasks = {
+  copySetup() {
```

Comment:
Tried to bundle all USWDS scaffolding tasks. Unsure of this name `copySetup()` it seems ambiguous.